SzczepionkaAPI/src/main/java/com/szczepionka/service/AppointmentService.java

```diff
@@ -33,9 +35,9 @@ public Appointment newAppointment(PatientDTO patientDTO, long locationId) {
         Patient patient = patientService.addPatient(patientDTO);
 
         Appointment appointment = Appointment.builder()
-                .id(patient.getId())
-                .appointmentDate(createAppointmentDate())
-                .appointmentTime(createAppointmentTime())
+                .id(patient.getId()) //FIXME patientID always 0
+                .appointmentDate(createAppointmentDate()) //TODO change from random to date selected by user
```

Comment:
I think that these comments are unnecessary too.  Create a card on Trello for this improvement/bug. (Mark there wherein code is the problem.)

Reply:
this TODO i can delete, as we spoke yesterday, it is already implemented correctly